### PR TITLE
Update release notes draft to version v1.26.0-alpha.1

### PR DESCRIPTION
**Pull request by harshanarayana**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.26.0-alpha.1

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

**Review by laxmikantbpandhare — APPROVED** [review]
/lgtm

**Review by AnaMMedina21 — APPROVED** [review]
/lgtm [forgot to approve earlier]

File: releases/release-1.26/release-notes/maps/pr-111808-map.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pr: 111808
+releasenote:
+  text: The errors in `k8s.io/apimachinery/pkg/api/meta` now support for the `stdlibs`
+    `errors.Is` matching, including when wrapped
```
Comment (ramrodo) [inline releases/release-1.26/release-notes/maps/pr-111808-map.yaml:4]:
```suggestion
    `errors.Is` matching, including when wrapped.
```

File: releases/release-1.26/release-notes/maps/pr-112150-map.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+pr: 112150
+releasenote:
+  text: Improved `kubectl` display of invalid request errors returned by the API server
```
Comment (ramrodo) [inline releases/release-1.26/release-notes/maps/pr-112150-map.yaml:3]:
```suggestion
  text: Improved `kubectl` display of invalid request errors returned by the API server.
```

File: releases/release-1.26/release-notes/maps/pr-112556-map.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+pr: 112556
+releasenote:
+  text: Changed error message when resource is not supported by given patch type in
+    `kubectl patch`
```
Comment (ramrodo) [inline releases/release-1.26/release-notes/maps/pr-112556-map.yaml:4]:
```suggestion
    `kubectl patch`.
```

File: releases/release-1.26/release-notes/maps/pr-112567-map.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+pr: 112567
+releasenote:
+  text: 'kube-scheduler: the `DefaultPodTopologySpread`, `NonPreemptingPriority`,
+    `PodAffinityNamespaceSelector` and `PreferNominatedNode` feature gates that graduated
+    to GA in v1.24 and were unconditionally enabled have been removed in v1.26'
```
Comment (ramrodo) [inline releases/release-1.26/release-notes/maps/pr-112567-map.yaml:5]:
```suggestion
    to GA in v1.24 and were unconditionally enabled have been removed in v1.26.'
```

**Review by ramrodo — COMMENTED** [review]
Good job!
Just some full stops missing

**Comment by harshanarayana** [comment]
@ramrodo  All the changes have been updated. Thanks for the comments. Missed them in a few files :)

**Comment by ramrodo** [comment]
Great! Thanks!
/lgtm
/approve

**Comment by ramrodo** [comment]
/assign @nate-double-u
Looking for an approval

**Comment by ramrodo** [comment]
cc @kubernetes/release-team-leads
Looking for an approval

**Pull request merged**